### DISCUSSION TITLE
I can no longer do business (buy) with "Bancos Nacionais (BR)" accounts

### DISCUSSION THREAD

**Discussion by Lexlanga** [bisq-network/bisq#7317] [Q&A]
Dear Sirs, could you please check why I can no longer do business (buy) with "Bancos Nacionais (BR)" accounts? Please note that I have already done business with them. My bank accounts are old. See attachment.

![print tela BISQ](https://github.com/user-attachments/assets/9759ead1-1a58-411d-9806-a2efed2bb7b6)

![image](https://github.com/user-attachments/assets/eeefeb2a-42ed-41a0-be0b-f6f50d4d1bbf)

**Comment by pazza83** [comment]
Is your account signed? All these offers for "Bancos Nacionais (BR)" are higher then 0.01 BTC so you would need a signed  "Bancos Nacionais (BR)" account to take the offers.
